### ISSUE TITLE
Tinkers lumberaxe didn't work like it should

### ISSUE THREAD

**Issue by MaGeBZz** [GTNewHorizons/GT-New-Horizons-Modpack#18103]
### Your GTNH Discord Username

magebzz_2

### Your Pack Version

2.7 Beta 4

### Your Server

Private Server

### Java Version

Java 17

### Type of Server

Don't know

### Your Expectation

If you break a log with the lumberaxe (unbreakable one in that case but should appear for all if they have enough durability) it should fell the whole tree or a whole row of trees if they are next to each other

### The Reality

It will work often but definetely not always
![image](https://github.com/user-attachments/assets/ac0430bc-ba68-4490-9448-63478d201c2d)
Like in this pic sometimes it will only be a 3x3 whatever why
Asked on the discord and another person confirmed it was so for a long time (and as long as I know minimum for 1.5 years) so would be very nice to get this fixed, very annoying cause if a tree is bugged like that and the whole tree won't be breaked then you can't even use it on the upper ones in the pic so the tree is basically "immune" to the lumberaxe

### Your Proposal

Being able to felling all the trees with a lumberaxe if there's enough durability or if it's unbreakable until you got at least saturation
Basically just like the way it is intended

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Miklebe** [comment]
I had no problems with cutting trees as long as they still had leaves/didn't have air gaps/i had enough hunger. How big is this plantation? Will try to replicate

**Comment by chochem** [comment]
also can't reproduce this.

**Comment by Miklebe** [comment]
Reproduced, but it's related to how the tree is generated, cut and checked. https://discord.com/channels/181078474394566657/522098956491030558/1310516919442341888 "detailed" description here.
In short - rainforest oak has a weird extension pattern 5x5 in a cross shape on top, the tree can connect via it and cut it of when chopping adjacent tree. Upon trying to cut the tree which lost its top, it's not recognized as a tree.

**Comment by chochem** [comment]
does not sound like a bug. just that you should space them out more
